shop3/docs/manual.texinfo

```diff
@@ -1023,25 +1025,33 @@ that variable substitutions in eval terms are handled before any
 evaluation of the expression, as in Lisp macros. One implication of this
 fact is that variables with symbolic values must be explicitly quoted if
 they are to be treated as Lisp symbols. For example, if the variable
-@code{?foo} is bound to the symbol @code{BAR}, the following eval term has the value
+@var{?foo} is bound to the symbol @code{BAR}, the following eval term has the value
```

Comment:
This shouldn't be redone this way, because `?foo` is not a lisp variable, it's a *shop* variable.